src/granite/transactions.cr

```diff
@@ -187,16 +185,6 @@ module Granite::Transactions
     save || raise Granite::RecordNotSaved.new(self.class.name, self)
   end
 
-  def update(*args, **named_args)
```

Comment:
We could probably support `#update` if we wanted.
1. Call `#initialize` with the values
2. Support only named arguments and do a loop with a case statement to set the ivar when there is a match